GTG/gtk/editor/editor.py

```diff
@@ -184,46 +194,113 @@ def __init__(self,
+    #     agr.connect(key, modifier, Gtk.AccelFlags.VISIBLE, self.close)
+
+    #     # F1 shows help
+    #     add_help_shortcut(self.window, "editor")
+
+    #     # Ctrl-N creates a new task
+    #     key, modifier = Gtk.accelerator_parse('<Control>n')
+    #     agr.connect(key, modifier, Gtk.AccelFlags.VISIBLE, self.new_task)
+
+    #     # Ctrl-Shift-N creates a new subtask
+    #     insert_subtask = self.builder.get_object("insert_subtask")
+    #     key, mod = Gtk.accelerator_parse("<Control><Shift>n")
+    #     insert_subtask.add_accelerator('clicked', agr, key, mod,
+    #                                    Gtk.AccelFlags.VISIBLE)
+
+    #     # Ctrl-D marks task as done
+    #     mark_as_done_editor = self.builder.get_object('mark_as_done_editor')
+    #     key, mod = Gtk.accelerator_parse('<Control>d')
+    #     mark_as_done_editor.add_accelerator('clicked', agr, key, mod,
+    #                                         Gtk.AccelFlags.VISIBLE)
+
+    #     # Ctrl-I marks task as dismissed
+    #     dismiss_editor = self.builder.get_object('dismiss_editor')
+    #     key, mod = Gtk.accelerator_parse('<Control>i')
+    #     dismiss_editor.add_accelerator('clicked', agr, key, mod,
+    #                                    Gtk.AccelFlags.VISIBLE)
+
+    #     # Ctrl-Q quits GTG
+    #     key, modifier = Gtk.accelerator_parse('<Control>q')
+    #     agr.connect(key, modifier, Gtk.AccelFlags.VISIBLE, self.quit)
+
+    '''
+    TODO(jakubbrindza): Add the functionality to the existing calendar widgets.
+    This will require ammending and re-factoring the entire calendar.py.
+    '''
+
+    def show_popover_start(self, widget, popover):
+        self.popover = self.builder.get_object("date_popover")
+        self.popover.set_relative_to(self.builder.get_object
+                                     ("startdate_entry"))
+        self.popover.set_modal(False)
+        # popover.hide(self.builder.get_object("fuzzy_btns"))
+        self.popover.show_all()
+
+    def show_popover_due(self, widget, popover):
+        self.popover = self.builder.get_object("date_popover")
+        self.popover.set_relative_to(self.builder.get_object("duedate_entry"))
+        self.popover.set_modal(False)
+        # popover.hide(self.builder.get_object("fuzzy_btns"))
+        self.popover.show_all()
+
+    def show_popover_closed(self, widget, popover):
+        self.popover = self.builder.get_object("closed_popover")
+        self.popover.set_relative_to(self.builder.get_object
+                                     ("closeddate_entry"))
+        self.popover.set_modal(False)
+        # popover.hide(self.builder.get_object("fuzzy_btns"))
+        self.popover.show_all()
+
+    def open_tags_popover(self, widget):
+        if self.store is None:
+            self.store = Gtk.ListStore(bool, str)
+            self.store.clear()
+
+            self.treeview = self.builder.get_object("tags_tree")
```

Comment:
You don't need to set it to the scope of class (it's used only in this method)

This is only an implementation detail, don't expose it to the outside world.